gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/OrchestratorTest.java

```diff
@@ -324,10 +323,10 @@ public void createFlowSpec() throws Throwable {
 
   /*
      If another flow has already acquired lease for this flowspec details within
-     epsilon time, then we do not execute this flow, hence do not process and store the spec
-     and throw LeaseUnavailableException
+     lease consolidation time, then we do not execute this flow, hence do not process and store the spec
+     and throw RuntimeException
    */
-  @Test(expectedExceptions = TooSoonToRerunSameFlowException.class)
+  @Test(expectedExceptions = RuntimeException.class)
```

Comment:
after signing off, I realized my literal advice would compromise clarity, foul up tests, etc. (in this very way)

sorry for that half-baked advice... try this instead:
```
public static class TooSoonToRerunSameFlowException extends RuntimeException {
  @Getter private final FlowSpec flowSpec;

  /**
   * Account for unwrapping within @{link FlowCatalog#updateOrAddSpecHelper}`s `CallbackResult` error handling for `SpecCatalogListener`s
   * @return `TooSoonToRerunSameFlowException` wrapped in another `TooSoonToRerunSameFlowException
   */
  public static TooSoonToRerunSameFlowException wrappedOnce(FlowSpec flowSpec) {
    return new TooSoonToRerunSameFlowException(flowSpec, new TooSoonToRerunSameFlowException(flowSpec));
  }

  public TooSoonToRerunSameFlowException(FlowSpec flowSpec) {
    this(flowSpec, null);
  }

  /** restricted-access ctor: use {@link #wrappedOnce(String)} instead */
  private TooSoonToRerunSameFlowException(FlowSpec flowSpec, Throwable cause) {
    super("Lease already occupied by another recent execution of this flow: " + flowSpec, cause);
    this.flowSpec = flowSpec;
  }
}
```

then replace:
```
throw new RuntimeException(new TooSoonToRerunSameFlowException(flowSpec));
```
with
```
throw TooSoonToRerunSameFlowException.wrappedOnce(flowSpec);
```